lib/puppet/module_tool/skeleton/templates/generator/examples/init.pp.erb

```diff
@@ -7,6 +7,6 @@
 # environment (to compare the resulting system state to the desired state).
 #
 # Learn more about module testing here:
-# https://docs.puppet.com/guides/tests_smoke.html
+# https://puppet.com/docs/puppet/latest/modules_fundamentals.html#writing-modules
```

Comment:
It wasn't immediately obvious where to go to read more about module testing when I followed the new link to the Modules Fundamentals doc. Would https://puppet.com/docs/puppet/5.5/bgtm.html#testing-your-module be a better link to use here? (I got to the Beginning Guide to Modules section from the links in the Modules Fundamentals doc, but only after clicking around hoping to find something that directly mentioned module testing.)

Reply:
Good call! I'll see if we can get that original URL redirected to the BGTM as well.